Formula/n/netcdf.rb

```diff
@@ -4,7 +4,7 @@ class Netcdf < Formula
   url "https://github.com/Unidata/netcdf-c/archive/refs/tags/v4.9.2.tar.gz"
   sha256 "bc104d101278c68b303359b3dc4192f81592ae8640f1aee486921138f7f88cb7"
   license "BSD-3-Clause"
-  revision 1
+  revision 2
```

Comment:
`vtk` failure looks due to `netcdf`. Seems like some bad linker flags are getting added to scripts that end up used by `vtk`.

Reply:
That's odd, I was able to build it locally. I will retry to see what is going on

Reply:
`netcdf` builds, but some of the files (maybe pkgconfig?) aren't correct which breaks `vtk` build

Spack just patches the generated files: https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/netcdf-c/package.py#L362

Reply:
I was able to fix it by only patching netCDFTargets.cmake. Let's see if it helps